tests/unit/h/schemas/api/group_test.py

```diff
@@ -13,167 +13,168 @@
-        # valid ``groupid``s — not as a length constraint
-        # Note also that the groupid does not have a valid authority but validation
-        # will raise on the formatting error before that becomes a problem.
-        with pytest.raises(ValidationError, match="groupid:.*does not match"):
-            schema.validate(
-                {
-                    "name": "Name not the Problem",
-                    "groupid": "group:"
-                    + ("o" * (AUTHORITY_PROVIDED_ID_MAX_LENGTH + 1))
-                    + "@foobar.com",
-                }
-            )
-
-    def test_it_raises_if_groupid_has_invalid_chars(self, schema):
-        with pytest.raises(ValidationError, match="groupid:.*does not match"):
-            schema.validate(
-                {"name": "Name not the Problem", "groupid": "group:&&?@thirdparty.com"}
-            )
-
-    def test_validate_raises_ValidationError_on_groupid_if_first_party(self, schema):
-        with pytest.raises(
-            ValidationError,
-            match="groupid may only be set on groups oustide of the default authority",
-        ):
-            schema.validate(
-                {"name": "Less Good", "groupid": "group:delicacy@hypothes.is"}
-            )
-
-    def test_validate_raises_ValidationError_if_no_group_authority(self):
-        schema = CreateGroupAPISchema(default_authority="hypothes.is")
-        with pytest.raises(
-            ValidationError,
-            match="groupid may only be set on groups oustide of the default authority",
-        ):
-            schema.validate(
-                {"name": "Blustery", "groupid": "group:delicacy@hypothes.is"}
-            )
-
-    def test_validate_raises_ValidationError_groupid_authority_mismatch(
-        self, third_party_schema
-    ):
-        with pytest.raises(ValidationError, match="Invalid authority.*in groupid"):
-            third_party_schema.validate(
-                {"name": "Shambles", "groupid": "group:valid_id@invalidauthority.com"}
-            )
+        with pytest.raises(ValidationError, match=error_message):
+            schema.validate(data)
 
-    @pytest.fixture
-    def schema(self):
+    def test_it_sets_authority_properties(self):
         schema = GroupAPISchema(
-            group_authority="hypothes.is", default_authority="hypothes.is"
+            group_authority="thirdparty.com", default_authority=DEFAULT_AUTHORITY
         )
-        return schema
 
-    @pytest.fixture
-    def third_party_schema(self):
-        schema = GroupAPISchema(
-            group_authority="thirdparty.com", default_authority="hypothes.is"
-        )
-        return schema
+        assert schema.group_authority == "thirdparty.com"
+        assert schema.default_authority == DEFAULT_AUTHORITY
 
 
 class TestCreateGroupAPISchema:
     def test_it_raises_if_name_missing(self, schema):
-        with pytest.raises(ValidationError, match=".*is a required property.*"):
```

Comment:
This weirdly tested that the error message includes `"is a required property"` but didn't test _which_ property the error message was saying was required!